model/AI/Properties/energyConsumption.md

```diff
@@ -8,7 +8,7 @@ Indicates the amount of energy consumed to build the AI package.
 
 ## Description
 
-EnergyConsumption captures the amount of energy needed to train and operate the AI model. 
+A free form text that captures the amount of energy needed to train and operate the AI model.
 This value is also known as training energy consumption or inference energy consumption.
```

Comment:
We should add definition of training and inference, ie. Training Energy Consumption is the amount of energy consumed during the training phase of an AI model. This includes the energy used to power the hardware and software systems used for training, such as GPUs, CPUs, and data center is
Inference Energy Consumption:is the amount of energy consumed during the inference phase of an AI model. This includes the energy used to power the hardware and software systems used for making predictions or decisions based on the model, such as edge devices, servers, or cloud services. Total Energy Consumption: is he total amount of energy consumed by an AI model over its entire lifecycle, including training, inference, and any other related activities

Reply:
Yes, your assumption is correct. This field (energyconsumption) of training data was added since IBM has a tool that calculates that metric.  (IBM Japan research demo'es this tool at Linux Summit 2022).  We can create a separate PR to discuss how to track/calculate total energy consumption, something that we need to think about is keeping AI BOM dynamic, AI systems are not statis.  A bigger discussion  But I approve your proposed description changes. .

Reply:
@bennetkl I have created #676 for the discussion of `inferenceEnergyConsumption` and `totalEnergyConsumption`